src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/model/OffenderEvent.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.justice.digital.hmpps.riskprofiler.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+/**
+ * A COPY of the custody-api class
```

Comment:
understand this is a copy.. maybe jackson's `JsonSubType`s would have been a nicer way to have modelled this originally